### ISSUE TITLE
[Request] 64-bit only buildbot checks

### ISSUE THREAD

**Issue by Schala** [msys2/MINGW-packages#7514]
Is it possible to have the build bot respect `arch=(x86_64)`? Right now, its logic is "I can't build on i686, this PR is no good." One of OpenAI's dependencies is a library called OneDNN that is 64-bit only. When trying to build with i686, it errors out, saying a 64-bit architecture is required.

**Comment by Biswa96** [comment]
There was a discussion about this. But for now the temporary workaround is to forcefully exit the i686 build. As an example, from [mingw-w64-openimagedenoise][1]

```sh
# XXX: skip 32bit build, no longer supported upstream
[[ $MINGW_CHOST = "i686-w64-mingw32" ]] && exit 0;
```

[1]: https://github.com/msys2/MINGW-packages/blob/master/mingw-w64-openimagedenoise/PKGBUILD

**Comment by Schala** [comment]
I see. Thanks

**Comment by Biswa96** [comment]
Also make sure to add the condition in the depends list in the dependent PKGBUILD file. Otherwise you'll get unresolved dependencies issue.

**Comment by Schala** [comment]
Well, buildbot just failed with this:

```
failure: <unknown> (mingw-w64-onednn): missing release number
	warning: <unknown> (mingw-w64-onednn): missing URL
	failure: <unknown> (mingw-w64-onednn): missing architecture
	failure: <unknown> (mingw-w64-onednn): missing license
	failure: <unknown> (mingw-w64-onednn): missing name
	failure: <unknown> (mingw-w64-onednn): missing version
	failure: <unknown> (mingw-w64-onednn): missing source
	warning: <unknown> (mingw-w64-onednn): missing description
```

**Comment by Biswa96** [comment]
Hence it is an workaround not a proper solution.

**Comment by MehdiChinoune** [comment]
Fixed by #10910
